scipy/stats/tests/test_generation/reference_distributions.py

```diff
@@ -333,6 +333,18 @@ def _pdf(self, x, alpha, beta):
         return a * q**-1 * mp.besselk(1, alpha*q) * mp.exp(beta*x)
 
 
+class Pearson3(ReferenceDistribution):
```

Comment:
The intent of the reference distribution is to be simple and easy to verify. I went ahead and pushed a simplification.

Reply:
Is this a variation of the formula for the pdf?

Reply:
It is the PDF as written in the docstring.

Reply:
Thank you.